packages/shared-ux/button/exit_full_screen/src/exit_full_screen_button.component.tsx

```diff
@@ -40,17 +44,23 @@ const description = i18n.translate(
  * A presentational component that renders a button designed to exit "full screen" mode.
  */
 export const ExitFullScreenButton = ({ onClick, className, customLogo }: Props) => {
-  const { euiTheme } = useEuiTheme();
-  const { colors, size, border } = euiTheme;
+  const euiThemeContext = useEuiTheme();
+  const { euiButtonDisplay } = euiButtonDisplayStyles(euiThemeContext);
```

Comment:
small note: I wanted to understand how this works and inspected the contents of `euiButtonDisplay`. I found it is an object that has a `styles` field, which is a string, and looks like this in my JS console:
```
> euiButtonDisplay.styles
'\n    display: inline-block;\n    appearance: none;\n    cursor: pointer;\n    [object Object];\n    white-space: nowrap;\n    max-inline-size: 100%;;\n    vertical-align: middle;\n   font-weight:500;\n    padding-block: 0;\n    padding-inline: 12px;\n     &:hover:not(:disabled),&:focus{text-decoration:underline;};label:euiButtonDisplay;'
```

I confirmed with @cee-chen that the `[object Object]` portion is not something to be concerned about.